src/SliderWidget.js

```diff
@@ -59,26 +72,29 @@ export default class SliderWidget extends Component {
   };
 
   okPress (valueSubmit) {
-
-    let notification = this.state.notification;
-
-    const notification_object = this.state.notificationObject;
-    const survey_object = this.state.notification.survey;
+    let notification_object = null;
+    if(this.state.notification){
+      let notification = this.state.notification;
+      notification_object = this.state.notificationObject;
+      
+    }
+    
+    const survey = this.state.survey;
     const location_object = this.state.locationObject;
 
     const id = uuid.v4();
     const answered_at = currentUnixTime;
     const sliderVal = this.state.metric;
     const app_version = this.state.app_version;
-    const user_id = notification.user_id;
+    
     const newSubmission = { 
       "id": id.toString(),
-      "user_id": user_id,
+      "user_id": this.state.user_id,
       "answered_at": answered_at,
       "wellness_value": sliderVal,
       "app_version": app_version,
       "notification": notification_object,
-      "survey": survey_object,
+      "survey": survey,
```

Comment:
related to my comment above, we probably want to distinguish between whether this is an object or not since we also have a survey array.